packages/deploy/deploy/300_catalyst/300_deploy_operator_filter_subscription.ts

```diff
@@ -7,22 +7,21 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const {deployer, sandAdmin} = await getNamedAccounts();
 
-  // TODO: review subscriptions for Catalyst and Asset
-
   // Operator filter subscription
-  await deploy('OperatorFilterSubscription', {
+
+  await deploy('OperatorFilterCatalystSubscription', {
```

Comment:
We have already deployed this on Polygon mainnet as `OperatorFilterSubscription`, was this captured by our versioning?

Reply:
Not sure I know what you mean, should renaming to `OperatorFilterSubscription ` to `OperatorFilterCatalystSubscription` in the deployments work?